uasc/secure_channel.go

```diff
@@ -81,6 +71,16 @@ type SecureChannel struct {
 	// may be calculated by adding the lifetime to the createdAt time.
 	lifetime uint32
 
+	// SecureChannelID is a unique identifier for the SecureChannel assigned by the Server.
```

Comment:
should be lowercase? also sequenceNumber?

Reply:
corrected